docs/orchestration/tutorial/overview.md

```diff
@@ -65,27 +65,39 @@ backend before you can proceed further.
 
 ### Create an API Key
 
-To authenticate, you'll need to create an [API Key](/orchestration/concepts/tokens.html#user) and configure it with the
-[Prefect Command Line Interface](/orchestration/concepts/cli.html#cli).
+To authenticate, you'll need to create an [API Key](/orchestration/concepts/tokens.html#user) and save it. 
 
 - Login to [https://cloud.prefect.io](https://cloud.prefect.io)
 - Navigate to the [API Keys page](https://cloud.prefect.io/user/keys). In the User menu in the top right corner go to **Account Settings** -> **API Keys** -> **Create An API Key**.
 - Copy the created key
-- Configure the CLI to use the key by running
+- Save the key locally either in
+your `~/.prefect/config.toml` config file, or as an environment variable:
```

Comment:
I think you'll want to combine this with the line above it for formatting purposes